lsix

```diff
@@ -274,6 +274,7 @@ main() {
 	done
 	montage "${onerow[@]}"  $imoptions gif:-  \
 	    | convert - -colors $numcolors sixel:-
+    echo
```

Comment:
I don't understand how this allows scrolling. Without it, I'm able to scroll back with <kbd>Shift</kbd>+<kbd>PgUp</kbd> using `xterm`. What terminal needs this?

Reply:
I use mlterm on ubuntu.
My use case implies full of jpeg. It seem's that without this 'echo', the terminal don't scroll after the last line of thumbs, and overwrite it.

Reply:
Hmm... I had previously added a special case for mlterm due to a sixel scrolling bug it had a couple years ago. Maybe they've fixed it and we simply need to remove that special case.

Reply:
I will search the history and test to remove it. I keep in touch